tests/staticfiles_tests/test_management.py

```diff
@@ -188,6 +188,27 @@ def _input(msg):
             return 'yes'
         return _input
 
+    def test_no_warning_when_staticdir_does_not_exist(self):
```

Comment:
I'd put these tests below the other cases since they aren't use the mock input method.